awscli/testutils.py

```diff
@@ -181,6 +181,45 @@ def create_bucket(session, name=None, region=None):
     return bucket_name
 
 
+def create_dir_bucket(session, name=None, location=None):
```

Comment:
It would be helpful to a developer to put a docstring for the type of `location`, since its a tuple.